Platform/TigerlakeBoardPkg/AcpiTables/Fadt/Fadt6.1.h

```diff
@@ -66,6 +70,9 @@
   EFI_ACPI_6_1_RTC_S4 | \
   EFI_ACPI_6_1_SLP_BUTTON | \
   EFI_ACPI_6_1_PROC_C1 | \
+  EFI_ACPI_6_1_LOW_POWER_S0_IDLE_CAPABLE | \
+  EFI_ACPI_6_1_PCI_EXP_WAK | \
+  EFI_ACPI_6_1_PWR_BUTTON | \

```

Comment:
Since S0ix config data is added, we need dynamic update FADT in PlatformUpdateAcpiTable() based on S0ix config data.

Reply:
Done in Stage2BoardInit.c, thanks.